department-of-veterans-affairs/caseflow
# Bump to Rails 5.2.4.6

### Description
Addresses CVE-2021-22885 and CVE-2021-22904

Strictly speaking we only need to bump the version on actionpack, but I'd rather not update that to 5.2.4.6 while keeping everything else at 5.2.4.5 if we can help it.

[Rails changelog](https://github.com/rails/rails/compare/v5.2.4.5...v5.2.4.6)

### Testing Plan
See [this Slack thread](https://dsva.slack.com/archives/C2ZAMLK88/p1589827462059100) for what was done last time, but to copy and paste here:

- Perform smoke test on Intake, Queue, Hearings in UAT.
  - Switch to an Intake user and start an intake.
  - Switch to a Queue user (VLJ) and load their queue.
  - Switch to a Hearings coordinator and load the ???
